### PR TITLE
Hide expired promos on initial load

### PR DESCRIPTION
**Pull request by itaisteinherz**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

This makes sure the expired promos are hidden when loading the website for the first time by hiding all promos by default, and then showing the valid promos using JS.
Note that I tried to do this by using a conditional expression in Pug where the promo gets the `"visible"` class if `swag.expired === false`, but it didn't work for some reason (probably because I don't know how to use Pug).

See more on this in #159 and https://github.com/swapagarwal/swag-for-dev/pull/159#issuecomment-457642144.

// cc @aslafy-z @vikaspotluri123

### CODE REVIEW

File: src/pug/includes/body.pug [hunk 1]
```diff
@@ -43,7 +43,7 @@ include ./fork.pug
 				- difficulty = swag.difficulty + ' difficulty';
 				- difficultyTitle = 'Difficulty: ' + swag.difficulty;
 				div(
-					class="item visible " + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
+					class="item " + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
```
Comment (itaisteinherz) [inline src/pug/includes/body.pug]: This also works:

```suggestion
					class="item " + (!swag.expired ? ' visible' : '') + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
```

But it seems like it doesn't make the promos load any faster (for me, at least).

**Comment by itaisteinherz** [comment]
@Yash-Garg You don't have to approve this (and other PRs) multiple times. Feel free to comment or suggest anything, but approving PRs multiple times doesn't help anyone.

File: src/pug/includes/body.pug (hunk 1 above)
Comment (vikaspotluri123) [reply src/pug/includes/body.pug]: I don't think we should worry too much about TTI in this specific case. We have a few other things to improve (most notably how we handle third party scripts) before we should handle stuff like this.

Disabling asset optimization on Netlify's side dropped TTI from 3.7 to 2.7 seconds for me so we're definitely not far from 🔥

**Comment by itaisteinherz** [comment]
@vikaspotluri123 I've added @aslafy-z's suggestion, and so I think this is ready to be merged. Also, I think #259 is ready as well.

File: src/pug/includes/body.pug [hunk 2]
```diff
@@ -42,8 +42,9 @@ include ./fork.pug
 			each swag in swagList
 				- difficulty = swag.difficulty + ' difficulty';
 				- difficultyTitle = 'Difficulty: ' + swag.difficulty;
+				- expired = swag.tags.includes('expired');
 				div(
-					class="item visible " + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
+					class="item " + (!expired ? 'visible ' : '') + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
```
Comment (vikaspotluri123) [inline src/pug/includes/body.pug]:
```suggestion
					class="item " + (expired ? '' : 'visible ') + swag.difficulty + swag.tags.map(t => ` tag-${t}`).join('')
```

**Comment by aslafy-z** [comment]
Ok, it works now with JS disabled. Thanks @itaisteinherz!
Let's merge!

**Pull request merged**